Docs: How to keep forked Eventum even with the upstream (#1222)
eventum/eventum: CONTRIBUTING.md

[good commit messages]: http://chris.beams.io/posts/git-commit/
[atomic commits]: http://www.freshconsulting.com/atomic-commits/

> Keep your forked repository even with the master one

Reviewer: ```suggestion
How to update forked repository to be up to date with the `master`:
```

- Check remotes `git remote -v`
- Add Eventum as upstream `git remote add upstream https://github.com/eventum/eventum.git`
- Switch to master `git checkout master`
- Fetch latest changes `git fetch upstream`

> - Rebase for clean pull requests `git rebase upstream/master`

Reviewer: ```suggestion
- Rebase for clean pull requests `git rebase -i upstream/master`
```

> - Update your forked repo `git push`

Reviewer: ```suggestion
```

pushing there will lead to errors

> - Do not work on `master` branch

Reviewer: ```suggestion
- Do not make commits on your `master` branch
```
